cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -931,6 +941,7 @@ construct_runtime!(
 		NftFractionalization: pallet_nft_fractionalization = 54,
 		PoolAssets: pallet_assets::<Instance3> = 55,
 		AssetConversion: pallet_asset_conversion = 56,
+		AssetsFreezer: pallet_assets_freezer::<Instance1> = 57,
```

Comment:
There're 3 instances of `pallet_assets`, so we'll need 3 instances of the freezer.